origin.spec

```diff
@@ -161,17 +168,16 @@ popd
 install -d %{buildroot}%{_bindir}
 
 # Install linux components
-for bin in openshift dockerregistry
+for bin in oc openshift dockerregistry
 do
   echo "+++ INSTALLING ${bin}"
   install -p -m 755 _build/bin/${bin} %{buildroot}%{_bindir}/${bin}
 done
 
 # Install client executable for windows and mac
-install -d %{buildroot}%{_datadir}/%{name}/{linux,macosx,windows}
-install -p -m 755 _build/bin/openshift %{buildroot}%{_datadir}/%{name}/linux/oc
-install -p -m 755 _build/bin/darwin_amd64/openshift %{buildroot}/%{_datadir}/%{name}/macosx/oc
-install -p -m 755 _build/bin/windows_386/openshift.exe %{buildroot}/%{_datadir}/%{name}/windows/oc.exe
+install -d %{buildroot}%{_datadir}/%{name}/{macosx,windows}
+install -p -m 755 _build/bin/darwin_amd64/oc %{buildroot}/%{_datadir}/%{name}/macosx/oc
+install -p -m 755 _build/bin/windows_386/oc.exe %{buildroot}/%{_datadir}/%{name}/windows/oc.exe
```

Comment:
I know it will bloat the re-distributable rpm but can we still add the linux binary to /usr/share/%{name}/linux in the clients-redistributable subpackage? Otherwise we have to ask RCM to get mac and windows binaries from one rpm and linux from another.